app/src/main/java/org/simple/clinic/bp/history/adapter/BloodPressureHistoryListItem.kt

```diff
@@ -38,29 +39,22 @@ sealed class BloodPressureHistoryListItem : PagingItemAdapter.Item<Event> {
     @SuppressLint("SetTextI18n")
     override fun render(holder: ViewHolderX, subject: Subject<Event>) {
       val context = holder.itemView.context
-      val bpDateTime = if (bpTime != null) {
-        context.getString(R.string.bloodpressurehistory_bp_time_date, bpDate, bpTime)
-      } else {
-        bpDate
-      }
-      val dateTimeTextAppearance = if (bpTime != null) {
-        TextAppearanceSpan(context, R.style.Clinic_V2_TextAppearance_Caption_Grey1)
-      } else {
-        TextAppearanceSpan(context, R.style.Clinic_V2_TextAppearance_Body2Left_Grey1)
-      }
-
```

Comment:
Something to keep a note of in future is that is ok (and preferable) to address Code Climate issues in a new commit since the previous commit is not actually broken, just could be improved.

Otherwise, this commit is doing two things:
- Refactoring some code
- Showing a different label for low BP

which should not be going together in a single commit.

Reply:
This is not necessary to fix in this current PR.

Reply:
Ah yes, I wanted to add it as a separate commit but wasn't sure about that. Will do that 👍

Reply:
Done. Moved it a different commit